src/sentry/api/endpoints/team_details.py

```diff
@@ -14,13 +15,23 @@
 from sentry.api.serializers import serialize
 from sentry.api.serializers.models.team import TeamSerializer as ModelTeamSerializer
 from sentry.api.serializers.rest_framework.base import CamelSnakeModelSerializer
+from sentry.apidocs.constants import (
+    RESPONSE_FORBIDDEN,
+    RESPONSE_NO_CONTENT,
+    RESPONSE_NOT_FOUND,
+    RESPONSE_UNAUTHORIZED,
+)
+from sentry.apidocs.examples.team_examples import TeamExamples
+from sentry.apidocs.parameters import GlobalParams, TeamParams
 from sentry.models.scheduledeletion import RegionScheduledDeletion
 from sentry.models.team import Team, TeamStatus
 
 
-class TeamSerializer(CamelSnakeModelSerializer):
+@extend_schema_serializer(exclude_fields=["name"])
+class TeamDetailsSerializer(CamelSnakeModelSerializer):
     slug = SentrySerializerSlugField(
         max_length=50,
+        help_text="Uniquely identifies a team and is used for the interface. Must be available.",
```

Comment:
```suggestion
        help_text="Unique identifier for a team. This is used for the ___ interface and must be available.",
```
Can you specify what interface it's used for?